src/main/groovy/com/avast/gradle/dockercompose/ComposeExtension.groovy

```diff
@@ -62,21 +63,25 @@ class ComposeExtension {
     }
 
     void exposeAsEnvironment(ProcessForkOptions task) {
-        servicesInfos.values().each { si ->
-            task.environment.put("${si.name.toUpperCase()}_HOST".toString(), si.host)
-            task.environment.put("${si.name.toUpperCase()}_CONTAINER_HOSTNAME".toString(), si.containerHostname)
-            si.tcpPorts.each {
-                task.environment.put("${si.name.toUpperCase()}_TCP_${it.key}".toString(), it.value)
+        servicesInfos.values().each { serviceInfo ->

```

Comment:
It would be great to make this change in both `exposeAs` methods: If the `instanceName` ends with `_1` then add envvar/sysprop that will have name without this suffix. So we will expose both `web_1` and `web`.